playbooks/site.yml

```diff
@@ -123,18 +124,25 @@
 
 - hosts: web
   roles:
-    - geerlingguy.mysql
     - chaos_jetzt.freescout
   tasks:
     - name: clone the httpheaderauth-module
+      become: yes
+      become_user: "{{ freescout_user }}"
       git:
         repo: 'https://github.com/chaos-jetzt/freescout-module-httpheaderauth.git'
         dest: "{{ freescout_basedir }}/Modules/HttpHeaderAuth"
+    - name: install the HttpHeaderAuth module
+      become: yes
+      become_user: "{{ freescout_user }}"
+      command:
+        cmd: "php artisan freescout:module-install httpheaderauth"
+        chdir: "{{ freescout_basedir }}"
     - name: enable the HttpHeaderAuth module
       become: yes
       become_user: "{{ freescout_user }}"
       command:
-        cmd: "php artisan module:enable HttpHeaderAuth"
+        cmd: "php artisan module:enable httpheaderauth"
```

Comment:
why is this now `httpheaderauth` instead of `HttpHeaderAuth`? or is it just case insensitive? If that's the case it might be good to change also line 134 to lowercase for consistency reasons

Reply:
As described in 43f165f2b850014bebf0b1eec00cb0bef4ef444a the issue is, that enabling the module fails if it's not all lower case. However, everything else (modules list, WebUI, ...) it's in the proper case.
I would leave it like that, since the name within the module.json:
https://github.com/chaos-jetzt/freescout-module-httpheaderauth/blob/b9f786059e788636ba64ecde3a099732a174813b/module.json#L2

Reply:
I go with you, you know more about freescout and php